pkg/store/mockstore/unistore/cophandler/mpp_exec.go

```diff
@@ -1075,9 +1075,13 @@ func (e *aggExec) processAllRows() (*chunk.Chunk, error) {
 
 	chk := chunk.NewChunkWithCapacity(e.fieldTypes, 0)
 
-	for i, gk := range e.groupKeys {
+	if len(e.groupKeys) == 0 {
+		// empty set, no data in agg OP.
+		// fill a mock Contexts here, to let agg get default aggregated result.
+		// like count(empty-set) = 0, sum(empty-set) = null
 		newRow := chunk.MutRowFromTypes(e.fieldTypes)
-		aggCtxs := e.getContexts(gk)
+		// empty context key
+		aggCtxs := e.getContexts([]byte{})
 		for i, agg := range e.aggExprs {
```

Comment:
maybe add a new function for this for loop

Reply:
make sense